symbols/overlay29.yml

```diff
@@ -8641,6 +8641,26 @@ overlay29:
         Called when the in-dungeon "others" menu is open. Does not return until the menu is closed.
         
         return: Always 0
+    - name: GetExclusiveItemWithEffectFromBag
```

Comment:
Can you fix the symbol ordering? Symbols are ordered by address.

Reply:
Is there a script to automate the ordering?

Reply:
Yes there is automation. See [the docs](https://github.com/UsernameFodder/pmdsky-debug/blob/master/symbols/README.md#follow-style-conventions)